.github/workflows/ci.yml

```diff
@@ -26,8 +26,11 @@ jobs:
         PYTHON:
           - {VERSION: "3.13", NOXSESSION: "flake"}
           - {VERSION: "3.13", NOXSESSION: "rust"}
+          - {VERSION: "3.13", NOXSESSION: "rust,tests", NOX_INSTALL_ARGS: "--features='pyo3/abi3-py37'"}
+          - {VERSION: "3.13", NOXSESSION: "rust,tests", NOX_INSTALL_ARGS: "--features='pyo3/abi3-py39'"}
+          - {VERSION: "3.13", NOXSESSION: "rust,tests", NOX_INSTALL_ARGS: "--features='pyo3/abi3-py311'"}
           - {VERSION: "3.12", NOXSESSION: "docs", OPENSSL: {TYPE: "openssl", VERSION: "3.5.0"}}
-          - {VERSION: "3.14-dev", NOXSESSION: "tests"}
+          - {VERSION: "3.14-dev", NOXSESSION: "tests", NOX_INSTALL_ARGS: "--features='pyo3/abi3-py313'"}
```

Comment:
Hmm, is this required because without it, the build fails?

Reply:
Yes since 3.14 is not a currently supported version by the production tooling

Reply:
This is one of those consequences of dropping abi3 by default -- with abi3 we can build fine against py314. (I should have included this in my original list)